deps/chakrashim/include/v8.h

```diff
@@ -2897,7 +2897,7 @@ bool PersistentBase<T>::IsNearDeath() const {
 
 template <class T>
 bool PersistentBase<T>::IsWeak() const {
-  return static_cast<bool>(_weakWrapper);
+  return _weakWrapper != nullptr;
```

Comment:
This was the cause of the C4800 warning.  Apparently VS2017 no longer issues a warning for it, but it seems weird to static cast when we're essentially just checking whether this is null?